app/models/conversion_host/configurations.rb

```diff
@@ -41,14 +41,22 @@ def enable(params)
       params = params.symbolize_keys
       _log.info("Enabling a conversion_host with parameters: #{params}")
 
-      params.delete(:task_id)     # In case this is being called through *_queue which will stick in a :task_id
-      params.delete(:miq_task_id) # The miq_queue.activate_miq_task will stick in a :miq_task_id
+      task_id = params.delete(:task_id)         # In case this is being called through *_queue which will stick in a :task_id
+      miq_task_id = params.delete(:miq_task_id) # The miq_queue.activate_miq_task will stick in a :miq_task_id
+      task_id ||= miq_task_id
 
       vddk_url = params.delete(:param_v2v_vddk_package_url)
 
       conversion_host = new(params)
       conversion_host.enable_conversion_host_role(vddk_url)
       conversion_host.save!
+
+      if task_id.present?
+        task = MiqTask.find(task_id)
+        task.options[:conversion_host_id] = conversion_host.id
```

Comment:
Is `options` right?  I thought it was `context`...

Reply:
We've used `options` so much that it sounded like the logical choice.
@djberg96 would you recommend `context_data` over `options` ?

Reply:
I mean is options even an attribute of an miq_task?

```
>> MiqTask.create!.options
NoMethodError (undefined method `options' for #<MiqTask:0x000055dc41439fe0>)
```

Reply:
@fdupont-redhat you're probably thinking of an `MiqRequestTask`, that has an `options` column but `MiqTask` does not.

Reply:
@agrare Now I'm curious what the difference between those two.

Reply:
They're different models?